src/uct/ib/base/ib_iface.c

```diff
@@ -1526,6 +1529,9 @@ ucs_status_t uct_ib_iface_query(uct_ib_iface_t *iface, size_t xport_hdr_len,
         return UCS_ERR_IO_ERROR;
     }
 
+    ucm_get_mem_type_current_device_info(UCS_MEMORY_TYPE_CUDA, &bus_id);
+    ucs_debug("bus_id: %x:%x:%x:%x \n", bus_id.domain, bus_id.bus, bus_id.slot, bus_id.function);
```

Comment:
I think we can remove this change from this PR and add it to the next PR where you use it.